scripts/deploy-production.js

```diff
@@ -14,113 +22,98 @@ const main = async () => {
     return;
   }
 
-  // Print all commits between the last tag and the current commit
   const commitMessages = await getCommitMessages();
-  let shouldDeployLocal = false;
+
+  // If there are no commit messages, ask the user if they want to delete and re-push the latest tag, thereby forcing a deploy with no new commits.
   if (commitMessages.length === 0) {
-    // If there are no commit messages, ask the user if they want to deploy anyway
     const response = await prompts({
       type: "confirm",
       name: "value",
-      message: "No new commits. Do you want to deploy anyway?",
+      message:
+        "No new commits. Do you want to deploy the most recent existing tag?",
       initial: false,
     });
-    shouldDeployLocal = response.value;
-    if (!shouldDeployLocal) {
+
+    const forceDeploy = response.value;
+    if (forceDeploy) {
+      await deleteAndPushLatestTag();
       return;
     }
-  } else {
-    console.log("Commit messages:");
-    console.log(commitMessages);
+
+    console.log(
+      "Deploy canceled. If systems are experiencing an outage and you'd like to push the deploy directly to S3, run yarn deploy:prod --local."
+    );
+    return;
   }
 
+  // Print all commits between the last tag and the current commit
+  console.log("Commit messages:\n" + commitMessages);
```

Comment:
My linter throws the following error 
![image](https://github.com/evergreen-ci/spruce/assets/4605522/80fdb688-6517-4318-949e-09a78eca3b78)

Reply:
This is one of the default rules on the eslint Airbnb rules which is what we have applied project-wide. I'm not sure why it didn't pick up for you. Maybe this folder isn't checked by our lint scripts? https://airbnb.io/javascript/#es6-template-literals

Reply:
Yep you were right we weren't looking in `scripts/`!